at_onboarding_flutter/lib/screen/at_onboarding_backup_screen.dart

```diff
@@ -44,7 +44,10 @@ class _AtOnboardingBackupScreenState extends State<AtOnboardingBackupScreen> {
       backgroundColor: Theme.of(context).scaffoldBackgroundColor,
       key: _scaffoldKey,
       appBar: AppBar(
-        title: const Text(AtOnboardingStrings.saveBackupKeyTitle),
+        title: const Text(
+          AtOnboardingStrings.saveBackupKeyTitle,
+          style: TextStyle(color: Colors.black),
```

Comment:
Don't set text color. Because we have a theme switch function and it will make text color constant when turn on dark mode.